nexus/db-lookup/src/lookup.rs

```diff
@@ -199,19 +199,9 @@ impl<'a> LookupPath<'a> {
 
     // Fleet-level resources
 
-    /// Select a resource of type ConsoleSession, identified by its `token`
-    pub fn console_session_token<'b, 'c>(
-        self,
-        token: &'b str,
-    ) -> ConsoleSession<'c>
-    where
-        'a: 'c,
-        'b: 'c,
-    {
-        ConsoleSession::PrimaryKey(
-            Root { lookup_root: self },
-            token.to_string(),
-        )
+    /// Select a resource of type ConsoleSession, identified by its `id`
+    pub fn console_session_id(self, id: Uuid) -> ConsoleSession<'a> {
+        ConsoleSession::PrimaryKey(Root { lookup_root: self }, id)
```

Comment:
These are not currently used, but they will be when we add endpoints to expire tokens and sessions by ID.

Reply:
I updated a test to use this.